nlmod/gwf/wells.py

```diff
@@ -199,11 +200,26 @@ def maw_from_df(
     perioddata = []
 
     iw = 0
-    for index, irow in df.iterrows():
+    for index, irow in tqdm(df.iterrows(), total=len(df), desc="Adding MAW wells"):
```

Comment:
maybe add a optional 'silent' argument to maw_from_df (with a default value of False), which is passed to tqdm as a 'disable` argument. I think we have similar uses in other parts of nlmod.